src/modules/blockchainApplication/explore/components/BlockchainApplications/WarnMissingAppMetaData.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useTranslation, withTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import { useTheme } from 'src/theme/Theme';
+import { SecondaryButton } from 'src/theme/buttons';
+import FlashMessage from 'src/theme/flashMessage/flashMessage';
+import Icon from 'src/theme/Icon';
+import styles from './WarnMissingAppMetaData.css';
+
+const WarnMissingAppMetaData = ({ registerApplication, ...props }) => {
+  const theme = useTheme();
+  const { t } = useTranslation();
+
+  return (
+    <FlashMessage shouldShow hasCloseAction={false} {...props} className={styles.flashContainer}>
+      <FlashMessage.Content>
+        <div className={`${styles.container} ${theme === 'dark' ? theme : ''}`}>
+          <Icon name="warningYellow" />
+          {t(
+            "Kindly provide your sidechain's off-chain metadata to empower sidechain users in seamlessly managing the application within their wallets."
+          )}
+          <div className={styles.btnContainer}>
+            <SecondaryButton
+              className={`${styles.button} ${theme === 'dark' ? theme : ''}`}
+              size="s"
+              onClick={registerApplication}
```

Comment:
Why not using a Link tag?

Reply:
I would have to re-style mean while the secondary button and the extra class already does the job.

Reply:
But the correct element is link. Gives much better user experience since hovering a link gives clues as well as accessibility concerns. 

Design somehow feel wrong if this looks like a button but behaves like a link taking you away from the application :/